src/queries/ReadOnly/GetAllGeneralCharacteristics.ts

```diff
@@ -0,0 +1,62 @@
+import { gql } from '@apollo/client';
+
+export default gql`
+  query GetAllGeneralCharacteristics($id: UUID!) {
+    modelPlan(id: $id) {
+      generalCharacteristics {
+        isNewModel
+        existingModel
```

Comment:
@patrickseguraoddball 
Could I do adjustments to the query in a separate NOREF PR since it will affect more queries than the scope of this particular ticket/PR?